src/frontend/validations.ts

```diff
@@ -3,10 +3,17 @@
 const domainRegexp = /^\w(?:[\w-]{0,61}\w)?$/;
 const jiraTopleveldomains = ["atlassian.net", "jira.com"];
 
-export const validJiraDomains = (jiraDomain: string, jiraTopleveldomain: string): boolean =>
-	!!jiraDomain && !!jiraTopleveldomain &&
-	domainRegexp.test(jiraDomain) &&
-	jiraTopleveldomains.includes(jiraTopleveldomain);
+export const validJiraDomains = (jiraDomain: string, jiraTopleveldomain: string): boolean => {
+	try {
+		return !!jiraDomain && !!jiraTopleveldomain &&
+			domainRegexp.test(jiraDomain) &&
+			jiraTopleveldomains.includes(jiraTopleveldomain) &&
+			!!new URL(jiraDomain);
```

Comment:
We needed to make sure it was a properly formatted URL as well, and not just have `atlassian.net` in it.